packages/web-comment/src/routes/comment-appeal/find-planning-appeal/index.njk

```diff
@@ -1,5 +1,5 @@
 {% extends "../../../index.njk" %}
-{% set title = "Enter appeal reference" %}
+{% set title = "Enter the appeal reference - Comment on a planning appeal - Welcome to GOV.UK" %}
```

Comment:
Is this suffix needed for every page?

Reply:
Good point, it seems like it probably should be but this is the only page Mo picked this defect up on. Do you know a smart nunjucksian way to put a postfix on titles?

Reply:
Honestly I'm a bit confused how this works at all - the govuk root template appears to use `pageTitle`. 

I guess if our index/main template were to set `pageTitle` by concatenating `title` and the `Comment on a ...` suffix?

Reply:
Looks like this was picked up on other tickets too, just not mentioned outside of the word doc attachment: https://pins-ds.atlassian.net/browse/AAPD-836

@Si3rr4wow are you happy to sort for all pages?